mirrord/layer/src/socket/hooks.rs

```diff
@@ -415,7 +415,7 @@ pub(super) unsafe extern "C" fn recvmsg_detour(
 
 /// Not a faithful reproduction of what [`libc::sendmsg`] is supposed to do, see [`sendmsg`].
 ///
-/// TODO(alex): We are ignoring the control message header [`libc::cmgshdr`].
+/// TODO(alex): We are ignoring the control message header [`libc::cmsghdr`].
```

Comment:
```suggestion
// TODO(alex): We are ignoring the control message header `libc::cmsghdr`.
```

Don't think this was supposed to be a doc.